examples/ui-demo/src/app/api/rpc/[...routes]/route.ts

```diff
@@ -2,6 +2,14 @@ import { NextResponse } from "next/server";
 
 import { env } from "../../../../../env.mjs";
 
+export const config = {
+  api: {
+    bodyParser: {
+      sizeLimit: "4mb",
```

Comment:
default is 1, just bumping this so we can bandaid this for a little until we gzip the cookies we generate